### PR TITLE
Updates python setup.py for recent license changes

### PR DESCRIPTION
**Pull request by perdasilva**
## Description ##
A recent PR #14726 has altered the way the static build deals with licenses. This has broken the `setup.py` script for building the release pip packages. This PR fixes this problem.

I've re-added some of the removed licenses, this should be confirmed to be OK by the submitters of the aforementioned PR. @lanking520 could you please take a look?

### CODE REVIEW

File: tools/staticbuild/build_lib.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ $MAKE DEPS_PATH=$DEPS_PATH $PWD/3rdparty/tvm/nnvm/lib/libnnvm.a
 $MAKE DEPS_PATH=$DEPS_PATH PSLITE
 
 if [[ $VARIANT == *mkl ]]; then
+    MKLDNN_LICENSE='license.txt'
```
Comment (zachgk) [inline tools/staticbuild/build_lib.sh]: These licenses are included as part of the source LICENSE. There is no need to re-add them

File: tools/pip/setup.py [hunk 2]
```diff
@@ -66,12 +66,8 @@ def has_ext_modules(self):
 shutil.copy(LIB_PATH[0], os.path.join(CURRENT_DIR, 'mxnet'))
 
 # copy license and notice
-shutil.copy(os.path.join(CURRENT_DIR, 'mxnet-build/LICENSE'),
-            os.path.join(CURRENT_DIR, 'mxnet/LICENSE'))
-shutil.copy(os.path.join(CURRENT_DIR, 'mxnet-build/DISCLAIMER'),
```
Comment (zachgk) [inline tools/pip/setup.py]: We do not currently add this DISCLAIMER. Can you add it to the build similarly to https://github.com/apache/incubator-mxnet/pull/14726/files#diff-2c9b07f53d24caa1ac38f1eb9c371118R70?

**Comment by perdasilva** [comment]
@zachgk I've addressed your comments ^^ thanks for taking a look!

**Comment by lanking520** [comment]
@szha

**Pull request merged**